### PR TITLE
Fix #1661, Fix #1662: Show graphic & icons when disabling Rewards AC/Ads

### LINKED ISSUE #1661: Disabling ads should show additional "Earnings are paid every month. Set your desired frequency to increase or decrease your earnings." Copy and icon

### LINKED ISSUE #1662: Disabling Auto-Contribute should show "Reward creators for the content you love. Your monthly payment gets distributed across the sites you visit." Copy and icon

### PR DESCRIPTION
**Pull request by kylehickinson**
Also updates Auto-Contribute copy to match desktop

@jamesmudgett (or @jenn-rhim) to verify design

## Summary of Changes

This pull request fixes issue #1661 
This pull request fixes issue #1662 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Test Plan:

- Enable rewards
- In rewards settings, turn off A/C & Ads
- Verify graphics + text appear in place of the view details button
- Change region to unsupported region
- Verify disabled graphic + text _does not_ appear for ads while it is unsupported

## Screenshots:

| Disabled | Unsupported |
| --- | --- |
| Simulator Screen Shot - iPhone 11 Pro - 2019-11-28 at 17 31 41 | Simulator Screen Shot - iPhone 11 Pro - 2019-11-28 at 17 31 01 |

## Reviewer Checklist:

- [ ] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: BraveRewardsUI/Settings/Ads/SettingsAdSectionView.swift [hunk 1]
```diff
@@ -130,3 +146,35 @@ class SettingsAdSectionView: SettingsSectionView {
     $0.font = SettingsUX.bodyFont
   }
 }
+
+private class DisabledAdsView: UIStackView {
+  
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    
+    axis = .horizontal
```
Comment (iccub) [inline BraveRewardsUI/Settings/Ads/SettingsAdSectionView.swift]: The axis is horizontal by default

File: BraveRewardsUI/Settings/Auto-Contribute/SettingsAutoContributeSectionView.swift [hunk 2]
```diff
@@ -70,3 +85,35 @@ class SettingsAutoContributeSectionView: SettingsSectionView {
     $0.font = SettingsUX.bodyFont
   }
 }
+
+private class DisabledAutoContributeView: UIStackView {
+  
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    
+    axis = .horizontal
```
Comment (iccub) [inline BraveRewardsUI/Settings/Auto-Contribute/SettingsAutoContributeSectionView.swift]: axis is horizontal by default

File: BraveRewardsUI/Settings/Auto-Contribute/SettingsAutoContributeSectionView.swift [hunk 3]
```diff
@@ -70,3 +85,35 @@ class SettingsAutoContributeSectionView: SettingsSectionView {
     $0.font = SettingsUX.bodyFont
   }
 }
+
+private class DisabledAutoContributeView: UIStackView {
```
Comment (iccub) [inline BraveRewardsUI/Settings/Auto-Contribute/SettingsAutoContributeSectionView.swift]: This and `DisabledAdsView` are very similar, wdyt about abstracting it?

**Review by iccub — CHANGES_REQUESTED** [review]

File: BraveRewardsUI/Settings/Ads/SettingsAdSectionView.swift [hunk 4]
```diff
@@ -130,3 +146,35 @@ class SettingsAdSectionView: SettingsSectionView {
...
+    spacing = 10
+    alignment = .center
+    layoutMargins = UIEdgeInsets(top: 15, left: 15, bottom: 15, right: 15)
```
Comment (Brandon-T) [inline BraveRewardsUI/Settings/Ads/SettingsAdSectionView.swift]: `UIEdgeInsets(equalInset: 15)`

File: BraveRewardsUI/Settings/Auto-Contribute/SettingsAutoContributeSectionView.swift [hunk 5]
```diff
@@ -70,3 +85,35 @@ class SettingsAutoContributeSectionView: SettingsSectionView {
...
+  
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    
+    axis = .horizontal
+    spacing = 10
+    alignment = .center
+    layoutMargins = UIEdgeInsets(top: 15, left: 15, bottom: 15, right: 15)
```
Comment (Brandon-T) [inline BraveRewardsUI/Settings/Auto-Contribute/SettingsAutoContributeSectionView.swift]: `UIEdgeInsets(equalInset: 15)`

**Review by Brandon-T — CHANGES_REQUESTED** [review]

File: BraveRewardsUI/Settings/Ads/SettingsAdSectionView.swift (hunk 4 above)
Comment (kylehickinson) [reply BraveRewardsUI/Settings/Ads/SettingsAdSectionView.swift]: Can't use these, they are in `Client`, and this is in the `BraveRewardsUI` framework. I can move `GeometryExtensions`  into BraveShared if you want?

**Review by iccub — APPROVED** [review]

**Review by Brandon-T — APPROVED** [review]

**Comment by kylehickinson** [comment]
New icons:

Simulator Screen Shot - iPhone 11 Pro - 2020-01-02 at 14 28 28

**Review by jamesmudgett — APPROVED** [review]

**Pull request merged**